liquibase/liquibase.github.com
# Update Master from Staging

## What I Did
I'd like to have the Installer blog go live. @SteveDonie - are these changes you have in Staging okay to go to Master, too? 

## How To Test It
Check to make sure everything that is currently in Staging is good to go to Master.
